### PR TITLE
add autoware_auto_perception_rviz_plugin README

### PR DESCRIPTION
**Pull request by taichiH**
## Description

Adding autoware_auto_perception_rviz_plugin README.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [pull request guidelines][pull-request-guidelines]
- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Review by yukkysaito — APPROVED** [review]

File: common/util/rviz_plugins/autoware_auto_perception_rviz_plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# autoware_auto_perception_plugin
+
+## Purpose
+
+It is a rviz plugin for visualizing the result from perception module.
```
Comment (kmiya) [inline common/util/rviz_plugins/autoware_auto_perception_rviz_plugin/README.md:5]:
```suggestion
It is an rviz plugin for visualizing the result from perception module.
```
Nit

File: common/util/rviz_plugins/autoware_auto_perception_rviz_plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+The visualization function developed in Autoware Architecture Proposal has been added to the implementation based rviz plugin in Autoware.Auto.
```
Comment (kmiya) [inline common/util/rviz_plugins/autoware_auto_perception_rviz_plugin/README.md:6]: Question: 日本語だとどういう文になりますか。Autoware.Autoのrvizプラグインを参考に実装したということでしょうか。

**Comment by kmiya** [comment]
Alas, has been merged.

File: common/util/rviz_plugins/autoware_auto_perception_rviz_plugin/README.md (hunk 2 above)
Comment (taichiH) [reply common/util/rviz_plugins/autoware_auto_perception_rviz_plugin/README.md:6]: すいません、おかしいので直します。

File: common/util/rviz_plugins/autoware_auto_perception_rviz_plugin/README.md (hunk 1 above)
Comment (taichiH) [reply common/util/rviz_plugins/autoware_auto_perception_rviz_plugin/README.md:5]: https://github.com/tier4/AutowareArchitectureProposal.iv/pull/634

File: common/util/rviz_plugins/autoware_auto_perception_rviz_plugin/README.md (hunk 2 above)
Comment (taichiH) [reply common/util/rviz_plugins/autoware_auto_perception_rviz_plugin/README.md:6]: https://github.com/tier4/AutowareArchitectureProposal.iv/pull/634

Comment (kmiya) [reply common/util/rviz_plugins/autoware_auto_perception_rviz_plugin/README.md:6]: :+1:

**Pull request merged**